lib/database/models/log.js

```diff
@@ -55,7 +55,6 @@ module.exports = (sequelize) => {
         Log.belongsToMany(models.Run, { as: 'runs', through: 'log_runs' });
         Log.belongsToMany(models.Tag, { as: 'tags', through: 'log_tags' });
         Log.belongsToMany(models.Environment, { as: 'environments', through: 'log_environments' });
-        Log.belongsToMany(models.Subsystem, { as: 'subsystems', through: 'log_subsystems' });
```

Comment:
Should the ENUM not be updated of type of log? Line28 `subsystem`?

Reply:
Done